contracts/governance/compatibility/GovernorCompatibilityBravo.sol

```diff
@@ -60,7 +62,13 @@ abstract contract GovernorCompatibilityBravo is IGovernorTimelock, IGovernorComp
     }
 
     /**
-     * @dev See {IGovernorCompatibilityBravo-propose}.
+     * @dev Alternative interface with separate function signatures. Stores the full proposal and fallback to the
```

Comment:
I'm not sure this should be user-facing docs. I think it should be more like a comment inside the function, would you agree?